content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm.md

```diff
@@ -135,7 +135,13 @@ To enable Single Step Instrumentation with the Datadog Operator:
      features:
        apm:
          instrumentation:
-           enabled: true  
+           enabled: true
+           libVersions:
+             java: v1
```

Comment:
I might be spacing out but what was the actual consensus on the whole `v` prefix?

Reply:
ah good catch. We can safely drop the v-prefix now.